### PR TITLE
feat: added the admin screen

### PR DESCRIPTION
**Pull request by AJcreativity001**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -32,6 +34,8 @@ export default function Router() {
           <Route path="jobs" element={<Jobs />} />
           <Route path="companies" element={<Companies />} />
           <Route path="forum" element={<Forum />} />
+          <Route path="admins" element={<Admins />} />
+          <Route path="admins/invite_new_admin" element={<InviteNewAdmin />} />
```
Comment (copy-ninja1) [inline src/App.tsx:38]:
```suggestion
          <Route path="admins/invite-new-admin" element={<InviteNewAdmin />} />
```

File: src/Modules/Admins/components/invite_new_admin.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import {
```
Comment (copy-ninja1) [inline src/Modules/Admins/components/invite_new_admin.tsx:1]: this should  be a page not a component

File: src/Modules/Admins/components/adminTable/admin_table_header.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      title: "Action",
+    },
+  ];
+  return (
+    <>
+      <Thead bgColor="primary" h={{ lg: "4.375rem" }}>
+        <Tr>
+          {tableHeaderTitle.map((t) => {
```
Comment (copy-ninja1) [inline src/Modules/Admins/components/adminTable/admin_table_header.tsx:22]:
```suggestion
          {tableHeaderTitle.map((tableData) => {
```

File: src/Modules/Admins/components/adminTable/admin_table_data.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+export interface IAdminTableResponsePayload {
```
Comment (copy-ninja1) [inline src/Modules/Admins/components/adminTable/admin_table_data.tsx:1]: move all the interface to 
_```src/Modules/Admins/components/adminTable/admin.interface.ts```_

File: package.json [hunk 5]
```diff
@@ -22,13 +22,15 @@
     "react-icons": "^4.8.0",
     "react-router-dom": "^6.10.0",
     "react-select": "^5.7.0",
+    "styled-components": "^5.3.10",
```
Comment (copy-ninja1) [inline package.json:25]: why styled-components

File: package.json [hunk 6]
```diff
@@ -22,13 +22,15 @@
...
     "yup": "^1.1.1"
   },
   "devDependencies": {
     "@faker-js/faker": "^7.6.0",
     "@types/node": "^18.15.11",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
+    "@types/styled-components": "^5.1.26",
```
Comment (copy-ninja1) [inline package.json:33]: why styled-components

File: src/Modules/Admins/components/invite_new_admin.tsx [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+              <Checkbox size="lg" />
+              <Text>Select all</Text>
+            </HStack>
+            <Box
+              mt={{ base: "1rem", md: "1.2rem", lg: "1.5rem" }}
+              display="flex"
+              flexDir="column"
+              gap="1rem"
+            >
+              {UsersPermissionData.map((p) => {
```
Comment (copy-ninja1) [inline src/Modules/Admins/components/invite_new_admin.tsx:75]:
```suggestion
              {UsersPermissionData.map((permission) => {
```

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (AJcreativity001) [reply package.json:25]: I needed it initially but no longer

File: src/App.tsx (hunk 1 above)
Comment (AJcreativity001) [reply src/App.tsx]: Okay

File: src/Modules/Admins/components/invite_new_admin.tsx (hunk 2 above)
Comment (AJcreativity001) [reply src/Modules/Admins/components/invite_new_admin.tsx]: Okay

File: src/Modules/Admins/components/adminTable/admin_table_data.tsx (hunk 4 above)
Comment (AJcreativity001) [reply src/Modules/Admins/components/adminTable/admin_table_data.tsx]: Okay sir

File: src/Modules/Admins/components/invite_new_admin.tsx (hunk 7 above)
Comment (AJcreativity001) [reply src/Modules/Admins/components/invite_new_admin.tsx]: okay sir

File: src/App.tsx [hunk 8]
```diff
@@ -16,6 +16,8 @@ import SettingsNotification from "@/Modules/Settings/Notification";
 import SettingsPasswordReset from "@/Modules/Settings/PasswordReset";
 
 import "@fontsource/inter";
+import Admins from "./Modules/Admins";
```
Comment (copy-ninja1) [inline src/App.tsx:19]: use 
```suggestion
import Admins from "@/Modules/Admins";
```

File: src/Modules/Admins/components/add_new_admin_button.tsx [hunk 9]
```diff
@@ -0,0 +1,33 @@
+import { Box, Button } from "@chakra-ui/react";
```
Comment (copy-ninja1) [inline src/Modules/Admins/components/add_new_admin_button.tsx:1]: use pascalCase for components and tsx file names 
eg : AddNewAdminButton

File: src/Modules/InviteNewAdmin/invite_new_admin.tsx [hunk 10]
```diff
@@ -0,0 +1,111 @@
+import {
```
Comment (copy-ninja1) [inline src/Modules/InviteNewAdmin/invite_new_admin.tsx:1]: make this page the index.tsx

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]

File: src/Modules/Admins/components/add_new_admin_button.tsx (hunk 9 above)
Comment (AJcreativity001) [reply src/Modules/Admins/components/add_new_admin_button.tsx:1]: Okay

File: src/Modules/InviteNewAdmin/invite_new_admin.tsx (hunk 10 above)
Comment (AJcreativity001) [reply src/Modules/InviteNewAdmin/invite_new_admin.tsx:1]: Okay sir

**Review by copy-ninja1 — APPROVED** [review]

**Pull request merged**